packages/cli/src/commands/dataMigrate/install.js

```diff
@@ -41,10 +41,14 @@ const appendModel = () => {
 
 // Create a new migration
 const save = async () => {
-  return await execa('yarn rw', ['db save', 'create data migrations'], {
-    cwd: getPaths().base,
-    shell: true,
-  })
+  return await execa(
+    'yarn rw',
+    ['prisma migrate dev --create-only', '--name=create_data_migrations'],
+    {
+      cwd: getPaths().base,
+      shell: true,
```

Comment:
Hey @dthyresson, adding `stdio: 'inherit'` here will prevent the issue happening incase these commands show up an interactive prompt

Reply:
Unfortunately that didn't help. Still same error.

In fact I get that error on every

`yarn rw prisma *`

now.

`yarn rw prisma studio`
`yarn rw prisma migrate dev`

Maybe I will wait until the other prisma command are merged in and try again.

Reply:
Yeah the other PR will fix the interactive issue. Just saying adding that here will also prevent future issues (if there's interactivity even with the flags)